api/SubtleCrypto.json

```diff
@@ -1158,6 +1158,43 @@
             "deprecated": false
           }
         },
+        "compressed_ec_points": {
+          "__compat": {
+            "description": "ECDSA or ECDH key data can contain compressed elliptic curve points.",
+            "support": {
+              "chrome": {
+                "version_added": "≤80"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "146"
+              },
+              "firefox_android": "mirror",
+              "nodejs": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": false
```

Comment:
Ran on latest safari 26 https://wpt.live/WebCryptoAPI/import_export/ec_importKey.https.any.html - the particular tests show up as optional and fail.